lights-out.js

```diff
@@ -16,4 +17,51 @@ class LightsOut {
             }
         }
     }
+
+    getNeighbors(row, col) {
+        const neighborArray = [];
+        
+        if(row < this.numRows - 1) {
+            if(row > 0) {
+                neighborArray.push([row + 1, col]);
+                neighborArray.push([row - 1, col]);
+            } else {
+                neighborArray.push([row + 1, col]);
+                neighborArray.push([this.numRows - 1, col]);
+            }

```

Comment:
the push `neighborArray.push([row + 1, col])` occurs the same regardless of the conditional (i.e. if (row > 0)

Therefore, you should move that push to above the conditional.